### PR TITLE
Update contribution guide

### PR DESCRIPTION
**Pull request by derberg**
Changes proposed in this pull request:

- Update contribution guide to point to more detailed info about how we work with GitHub issues
- Describe issues handling workflow

### CODE REVIEW

File: issues-workflow.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Backlog
+
+Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except of labels `question`). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning critical label. Critical issues assigned to the current milestone have the highest priority.
+
+>**NOTE** From the main Kyma backlog issues are taken by different teams that are responsible for specific areas of the Kyma. This is how we work on Kyma at the moment and this is why we started using GitHub milestone as an attribute to distinguish which team works on a specific issue. Thanks to this approach we can easily work in team sprints using ZenHub board. We are aware that our teams names are criptic for ya but unfortunately at the moment we were not able to come up with any better solution. If you have better ideas, please give us feedback. Sorry for the inconvenience. 
+
+### Team sprints
+Team Sprint is modeled as a GitHub milestone named with the following pattern: `{TEAM_NAME} Sprint {NUMBER}`. During the planing team selects issues from the backlog considering:
```
Comment (klaudiagrz) [inline issues-workflow.md]: `During the planning, a team`...

File: issues-workflow.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- priority (high priority first)
+- area (default capability of the team first)
+- dependencies (unblock others)
+
+Developers mark issues with WIP label when they start work on it. When the work is done issue is closed.
```
Comment (klaudiagrz) [inline issues-workflow.md]: Only developers use this mark? 😉 

1. ... ``with the `WIP` label when they start working on it. When the work is done, the issue is closed.``

File: issues-workflow.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Team backlog (optional)
+If the team wants to keep team backlog (assign issues they want to work on later) they can assign it to the Github milestone with no due date named with following pattern `{TEAM_NAME} backlog`
```
Comment (klaudiagrz) [inline issues-workflow.md]: `If any team wants to keep team backlog (assign issues they want to work on later), they can`...

Add a dot at the end of the sentence.

File: issues-workflow.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+This document explains how the issues/pull requests workflow is organized in Kyma project:
+- How issues triage is organized
+- What tools are used in the workflow on what stage
+
+## Used labels
+
+Our statement is to:
+* Use default labels provided by GitHub
+* Introduce new labels only if it is really needed
```
[the remark this answers is not in the document]
Comment (derberg) [reply issues-workflow.md]: old habits of old man

File: issues-workflow.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+| Priority | Prioritize issues in the general Kyma backlog to select those which are the most critical and should be taken as first. | critical |
+| Area | Clarify which capabilities are involved in a given issue. | area/{CAPABILITY_NAME} |
+
+## Backlog
+
+The Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except from the issues with `question` label). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning the `critical` label. Critical issues assigned to the current milestone have the highest priority.
+
+>**NOTE** Issues are taken from the main Kyma backlog by different teams that are responsible for specific areas of Kyma. This is the actual workflow in Kyma which allows distinguishing which team works on a specific issue. This approach allows easy work in team sprints using ZenHub board. We are aware that our teams names are criptic for ya but unfortunately at the moment we were not able to come up with any better solution. If you have better ideas, please give us feedback. Sorry for the inconvenience.
```
Comment (klaudiagrz) [inline issues-workflow.md]: 1. ``>**NOTE:**``  -> add " : "
2. ``We are aware that our teams' names may be cryptic for the external users, but at the moment we are not able to provide any better solution. If you have a better idea, your feedback would be highly appreciated. Sorry for the inconvenience.``

File: issues-workflow.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+| Area | Clarify which capabilities are involved in a given issue. | area/{CAPABILITY_NAME} |
+
+## Backlog
+
+The Kyma backlog contains issues that went through the triage, are not closed, and have labels added (except from the issues with `question` label). Backlog prioritization is realized by assigning issues to Kyma milestones (ZenHub Release) and assigning the `critical` label. Critical issues assigned to the current milestone have the highest priority.
+
+>**NOTE** Issues are taken from the main Kyma backlog by different teams that are responsible for specific areas of Kyma. This is the actual workflow in Kyma which allows distinguishing which team works on a specific issue. This approach allows easy work in team sprints using ZenHub board. We are aware that our teams names are criptic for ya but unfortunately at the moment we were not able to come up with any better solution. If you have better ideas, please give us feedback. Sorry for the inconvenience. 
+
+### Team sprints
+Team Sprint is modeled as a GitHub milestone named with the following pattern: `{TEAM_NAME} Sprint {NUMBER}`. During the planing a team selects issues from the backlog considering:
```
Comment (klaudiagrz) [inline issues-workflow.md]: ``During the planning, `` -> add a comma and double "n" in "planning"

File: issues-workflow.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- priority (high priority first)
+- area (default capability of the team first)
+- dependencies (unblock others)
+
+Contributors mark issues with WIP with the `WIP` label when they start working on it. When the work is done, the issue is closed.
```
Comment (klaudiagrz) [inline issues-workflow.md]: `Contributors mark issues` ~with WIP~ ``with the `WIP` label when they start working on it. When the work is done, the issue is closed.``

File: issues-workflow.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+### Team backlog (optional)
+If any team wants to keep team backlog (assign issues they want to work on later) they can assign it to the Github milestone with no due date named with following pattern `{TEAM_NAME} backlog`.
```
Comment (klaudiagrz) [inline issues-workflow.md]: 1. Please add a comma after " ) "
2. `named with the following pattern:`

File: issues-workflow.md (hunk 5 above)
Comment (derberg) [reply issues-workflow.md]: I just changed users to community

File: issues-workflow.md (hunk 7 above)
Comment (derberg) [reply issues-workflow.md]: 🤦‍♂️

**Pull request merged**